tests/behat/features/bootstrap/ApiFeatureContext.php

```diff
@@ -1114,6 +1114,34 @@ private function assertJsonRegex($pattern, $json) {
     $json = json_encode(json_decode($json));
     Assert::assertRegExp($delimter . $pattern . $delimter, $json);
   }
+  /**
+   * @Then /^I should get (\d+) programs in the following order:$/
+   * @param  $program_count
+
+   * @param TableNode $table
+   */
+  public function iShouldGetScratchProgramsInTheFollowingOrder($program_count, TableNode $table)
+  {
+    $response = $this->getClient()->getResponse();
+
+    $responseArray = json_decode($response->getContent(), true);
+    $programs = $table->getHash();
+
+    $returned_programs = $responseArray['CatrobatProjects'];
+    $scratch_programs_count = count($programs);
+    Assert::assertEquals($program_count, $scratch_programs_count, 'Wrong number of Scratch programs');
+    print($scratch_programs_count);
+    $expected_programs = $table->getHash();
+
+    for ($i = 0; $i < count($returned_programs); ++$i)
+    {
+      Assert::assertEquals(
+        $expected_programs[$i]['Name'], $returned_programs[$i]['ProjectName'],
+        'Wrong order of results'
+      );
+    }
+  }
+
```

Comment:
I'm pretty sure this function needs a slight rework. 
look at my comment @/get_scratch_remix_programs.feature